fatiando/mesher.py

```diff
@@ -1721,3 +1721,594 @@ def keep(cell):
+        ...                       strike=10, dip=20, rake=30, props={
+        ...                       'remanent magnetization': [10, 25, 40],
+        ...                       'susceptibility tensor': [0.562, 0.485,
+        ...                                                 0.25, 90, 0,
+        ...                                                 0]})
+        >>> e.props['remanent magnetization']
+        [10, 25, 40]
+        >>> e.addprop('density', 20)
+        >>> print(e.props['density'])
+        20
+        >>> print(e)
+        x:1 | y:2 | z:3 | large_axis:6 | intermediate_axis:5 | small_axis:4 | \
+strike:10 | dip:20 | rake:30 | density:20 | remanent magnetization:[10, 25, 40\
+] | susceptibility tensor:[0.562, 0.485, 0.25, 90, 0, 0]
+        >>> e = TriaxialEllipsoid(1, 2.7, 3, 6, 5, 4, 10, 20, 30)
+        >>> print(e)
+        x:1 | y:2.7 | z:3 | large_axis:6 | intermediate_axis:5 | small_axis:4 \
+| strike:10 | dip:20 | rake:30
+        >>> e.addprop('density', 2670)
+        >>> print(e)
+        x:1 | y:2.7 | z:3 | large_axis:6 | intermediate_axis:5 | small_axis:4 \
+| strike:10 | dip:20 | rake:30 | density:2670
+
+    """
+
+    def __init__(self, x, y, z, large_axis, intermediate_axis, small_axis,
+                 strike, dip, rake, props=None):
+        GeometricElement.__init__(self, props)
+
+        self.x = x
+        self.y = y
+        self.z = z
+        self.large_axis = large_axis
+        self.intermediate_axis = intermediate_axis
+        self.small_axis = small_axis
+        self.strike = strike
+        self.dip = dip
+        self.rake = rake
+
+        assert self.large_axis > self.intermediate_axis and \
+            self.intermediate_axis > self.small_axis, "large_axis must be grea\
+ter than intermediate_axis and intermediate_axis must greater than small_axis"
+
+        # Auxiliary orientation angles
+        self.alpha, self.gamma, self.delta = auxiliary_angles(self.strike,
+                                                              self.dip,
+                                                              self.rake)
+
+        # Coordinate transformation matrix
+        self.transf_matrix = coord_transf_matrix_triaxial(self.alpha,
+                                                          self.gamma,
+                                                          self.delta)
+
+    def __str__(self):
+        """
+        Return a string representation of the triaxial ellipsoid.
+        """
+
+        names = [('x', self.x), ('y', self.y), ('z', self.z),
+                 ('large_axis', self.large_axis),
+                 ('intermediate_axis', self.intermediate_axis),
+                 ('small_axis', self.small_axis), ('strike', self.strike),
+                 ('dip', self.dip), ('rake', self.rake)]
+        names = names + [(p, self.props[p]) for p in sorted(self.props)]
+        return ' | '.join('%s:%s' % (n, v) for n, v in names)
+
+    def susceptibility_tensor(self):
```

Comment:
I'd prefer this to be a property. If you add `@property` above the `def` it will turn this method into a property. The  you can access the tensor without the `()` like `ellipsoid.susceptibility_tensor` and it calls the method behind the scenes.